working dir/widgets/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+
+//buildDir = rootDir.resolve("build").resolve("widgets")
```

Comment:
I think the idea of separate build is misguided here. Plus the widget directory may change or split into multiple.

Reply:
it isn't a separate build, but a separate module of the project. I will definitely work on separating the widgets in the future, but for now it's important to get stuff working as wel as before

Reply:
Ok.